Documentation/gettingstarted/istio.rst

```diff
@@ -532,12 +541,34 @@ Check the progress of the deployment (every service should have an
 
     $ kubectl get deployments -n default
     NAME                  DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-    authaudit-logger-v1   1         1         1            1           23s
-    details-v1            1         1         1            1           16m
-    productpage-v2        1         1         1            1           2m
-    ratings-v1            1         1         1            1           11m
-    reviews-v1            1         1         1            1           16m
-    reviews-v2            1         1         1            1           11m
+    authaudit-logger-v1   1         1         1            1           20s
+    details-v1            1         1         1            1           22m
+    productpage-v2        1         1         1            1           4m
+    ratings-v1            1         1         1            1           19m
+    reviews-v1            1         1         1            1           22m
+    reviews-v2            1         1         1            1           19m
+
+Check that the product REST API is still accessible, and that Cilium
+now denies at Layer-7 any access to the reviews and ratings REST API:
+
+::
+
+    $ export PRODUCTPAGE=`minikube service istio-ingress -n istio-system --url | head -n 1`
+    $ for APIPATH in /api/v1/products /api/v1/products/0 /api/v1/products/0/reviews /api/v1/products/0/ratings; do echo ; curl -s -S "${PRODUCTPAGE}${APIPATH}" ; echo ; done
+
+    [{"descriptionHtml": "<a href=\"https://en.wikipedia.org/wiki/The_Comedy_of_Errors\">Wikipedia Summary</a>: The Comedy of Errors is one of <b>William Shakespeare's</b> early plays. It is his shortest and one of his most farcical comedies, with a major part of the humour coming from slapstick and mistaken identity, in addition to puns and word play.", "id": 0, "title": "The Comedy of Errors"}]
+
+    {"publisher": "PublisherA", "language": "English", "author": "William Shakespeare", "id": 0, "ISBN-10": "1234567890", "ISBN-13": "123-1234567890", "year": 1595, "type": "paperback", "pages": 200}
+
+    Access denied
+
+
+    Access denied
+
+This demonstrated that requests to the
+``/api/v1/products/<id>/reviews`` and
+``/api/v1/products/<id>/ratings`` URIs now result in Cilium returning
+``HTTP 403 Forbidden`` HTTP responses.
 
 Every login and logout on the product page will result in a line in
 this service's log:
```

Comment:
On line 573 I don't see anything, I was expecting the command to success since I do **not** have a kafka policy installed.

```
kubectl exec ${POD_LOGGER_V1} -c authaudit-logger -ti -- /opt/kafka_2.11-0.10.1.0/bin/kafka-console-producer.sh --broker-list=kafka:9092 --topic=authaudit
```

Reply:
That's because of the productpage-v2 Kafka connection issue. That's an application issue.